### PR TITLE
chore: move CM admin to own package

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* moves the CM admin back to it's plugin
* moves review-workflows to be a plugin (still WIP)

### Why is it needed?

* we've been wanting to do this for ages.

### Related issue(s)/PR(s)

* resolves CONTENT-2080

### CODE REVIEW

**Comment by joshuaellis** [comment]
This PR mainly just moves the CM, review-workflows isn't enabled anymore as we refactor that in separate work. It has been published to `npm` under `0.0.1` for now so it won't break anything in the meantime. If you use EE, it won't show in the menu either.

**Review by Marc-Roig — APPROVED** [review]

File: packages/core/content-releases/admin/src/index.ts [hunk 1]
```diff
@@ -28,14 +29,15 @@ const admin: Plugin.Config.AdminInput = {
        * For some reason every middleware you pass has to a function
        * that returns the actual middleware. It's annoying but no one knows why....
        */
+      // @ts-expect-error – this API needs to be styped better.
```
Comment (jhoward1994) [inline packages/core/content-releases/admin/src/index.ts:32]:
```suggestion
      // @ts-expect-error – this API needs to be typed better.
```

**Comment by jhoward1994** [comment]
image

I get this error when running getstarted in CE or EE

**Comment by joshuaellis** [comment]
> I get this error when running getstarted in CE or EE

did you run `yarn` and `yarn build`? 🤔

**Comment by joshuaellis** [comment]
Because the e2e tests do not fail, and they would for the error you're seeing im gonna merge 🚂

**Pull request merged**